Readme.md

```diff
@@ -33,6 +33,12 @@ The `-SettingsPath` can be omitted, in this case the _PSScriptAnalyzerSettings.p
 
 #### GitHub Actions
 
+If you are using our `build-dotnet` action or build-related reusable workflows from [Lombiq GitHub Actions](https://github.com/Lombiq/GitHub-Actions), PowerShell linting is already included: Just set the value of the `powershell-analyzer-path` to the path of the _Invoke-Analyzer.ps1_ file relative to your repository root. In case it's _./tools/Lombiq.Analyzers.PowerShell/Invoke-Analyzer.ps1_ you don't need additional configuration.
```

Comment:
I moved this up to be placed above the new instructions I just added.
@DAud-IcI what/where is the mechanism behind these? I couldn’t find anything related to this it in LGHA.

Reply:
I don't remember. I was suspicious that maybe it was removed so I went back to the original issue (OSOE-150) and didn't find `powershell-analyzer-path` there either. Let's remove this paragraph and the "Otherwise" from the next.